.github/actions/turbo-diffs/action.yml

```diff
@@ -13,6 +13,9 @@ runs:
     - uses: pnpm/action-setup@a3252b78c470c02df07e9d59298aecedc3ccdd6d # pin@v3.0.0
       with:
         version: 9.1.1
+    - name: Install dependencies
+      run: pnpm install --frozen-lockfile
+      shell: bash
```

Comment:
it was defaulting to using a new version of turbo that removed commands this job depends on, so e2e test were only triggered on rust changes